cola_inference.py

```diff
@@ -212,5 +254,40 @@ def take_one(ls):
     return ls[0]
 
 
+def load_cola_data(input_path, task, input_format, max_seq_len):
```

Comment:
Add a comment on why you need this here, rather than using the existing jiant loader.

Reply:
The current jiant loader leads data for all three phases at once, hence it is not possible to separate that functionality:

https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/tasks/tasks.py#L354-L359

Once possibility would be to expose e.g. `get_train`/`get_dev`/`get_test` methods specifically for the CoLA Task class. This can be done cleanly so as to not affect the regular usage of the CoLA Task/data loading or the broader Task api.

Reply:
Hrm. I guess we can merge what you have now, but create an issue to clean this up.